src/test/java/homework_2/random_chars_table/RandomCharsTableTest.java

```diff
@@ -0,0 +1,133 @@
+package homework_2.random_chars_table;
+
+import base.UnitBase;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RandomCharsTableTest extends UnitBase {
```

Comment:
Tests are failing
![image](https://user-images.githubusercontent.com/58872509/134571782-f6b5c1b8-a58d-4244-8dda-5865d786497a.png)

Reply:
Forgot to refactor test after app refactoring. Updated